common.sh

```diff
@@ -17,6 +17,8 @@ AFL_SRC=$(dirname $(dirname $SCRIPT_DIR))/AFL
 FUZZ_CXXFLAGS="-O2 -fno-omit-frame-pointer -g -fsanitize=address -fsanitize-coverage=trace-pc-guard,trace-cmp,trace-gep,trace-div"
 CORPUS=CORPUS-$EXECUTABLE_NAME_BASE
 JOBS=8
+# Every "gcloud compute ssh/scp/etc" needs --zone, otherwise throws warning
+GCLOUD_ZONE="us-west1-b"
```

Comment:
This could be considered as another feature which is unique to our group's setup. For any group, however, it is a value which will be needed in every script which calls `gcloud` (pretty much every script, likely including the workers). Moreover, it should probably be constant among dispatchers, workers, and storage buckets (also not yet implemented) for bandwidth efficiency/cheapness.

I'm skeptical of the alternative of defining it in each script, likely four different scripts.